product/urls.py

```diff
@@ -1,3 +1,7 @@
+from django.urls   import path
+
+from product.views import ProductListView
+
 urlpatterns = [
-    
-]
\ No newline at end of file
+    path('/list/', ProductListView.as_view()),
```

Comment:
```suggestion
    path('', ProductListView.as_view()),
```
8000/products 가 products의 자원을 가리키기 때문에 다음과 같이 수정하면 좋을 것 같습니다.